addons/crm/tests/common.py

```diff
@@ -358,11 +359,15 @@ def assertLeadMerged(self, opportunity, leads, **expected):
         :param opportunity: final opportunity
         :param leads: merged leads (including opportunity)
         """
+        # import pdb;
+        # pdb.set_trace()
         self.assertIn(opportunity, leads)
-
         # save opportunity value before being modified by merge process
-        fields_all = self.FIELDS_FIRST_SET + ['description', 'type', 'priority']
+        fields_all = self.FIELDS_FIRST_SET + self.merge_fields
```

Comment:
Seems correct indeed. Actually this test is not really important but as it is present let us keep it.